code/game/objects/items/oddities.dm

```diff
@@ -26,6 +27,14 @@
 		for(var/stat in oddity_stats)
 			oddity_stats[stat] = rand(1, oddity_stats[stat])
 	AddComponent(/datum/component/inspiration, oddity_stats)
+	if(!perk)
```

Comment:
Not part of a review, just a comment; I'm under the impression that A: Not every oddity needs a perk and B: You should theme the perk to the oddity.

Reply:
> Not part of a review, just a comment; I'm under the impression that A: Not every oddity needs a perk and B: You should theme the perk to the oddity.

I had a code to enable this.
![image](https://user-images.githubusercontent.com/49106866/86023259-71796d80-b9f9-11ea-9b24-ee6b1bd72490.png)
![image](https://user-images.githubusercontent.com/49106866/86023577-db921280-b9f9-11ea-8387-7e74ec944284.png)

Reply:
well, we can handle it in another PR then